apisix/plugins/serverless/generic-upstream.lua

```diff
@@ -35,6 +35,7 @@ return function(plugin_name, version, priority, request_processor, authz_schema,
             function_uri = {type = "string"},
             authorization = authz_schema,
             timeout = {type = "integer", minimum = 100, default = 3000},
+            use_default_user_agent = {type = "boolean", default = true},
```

Comment:
I think we can set the default value to false, which can reduce the cost of users.